annotator/src/test/java/org/cbioportal/models/MutationRecordTest.java

```diff
@@ -1,203 +1,36 @@
-/*
- * Copyright (c) 2019 Memorial Sloan-Kettering Cancer Center.
- *
- * This library is distributed in the hope that it will be useful, but WITHOUT
- * ANY WARRANTY, WITHOUT EVEN THE IMPLIED WARRANTY OF MERCHANTABILITY OR FITNESS
- * FOR A PARTICULAR PURPOSE. The software and documentation provided hereunder
- * is on an "as is" basis, and Memorial Sloan-Kettering Cancer Center has no
- * obligations to provide maintenance, support, updates, enhancements or
- * modifications. In no event shall Memorial Sloan-Kettering Cancer Center be
- * liable to any party for direct, indirect, special, incidental or
- * consequential damages, including lost profits, arising out of the use of this
- * software and its documentation, even if Memorial Sloan-Kettering Cancer
- * Center has been advised of the possibility of such damage.
- */
+package org.cbioportal.models;
 
-/*
- * This file is part of cBioPortal CMO-Pipelines.
- *
- * cBioPortal is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as
- * published by the Free Software Foundation, either version 3 of the
- * License.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU Affero General Public License for more details.
- *
- * You should have received a copy of the GNU Affero General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
-*/
+import org.junit.jupiter.api.Test;
 
-package org.cbioportal.model;
+import java.util.List;
 
-import java.lang.reflect.InvocationTargetException;
-import java.lang.reflect.Method;
-import java.util.*;
-import org.cbioportal.models.AnnotatedRecord;
-import org.cbioportal.models.MutationRecord;
-import org.junit.*;
-import org.junit.runner.RunWith;
-import org.cbioportal.annotator.MockGenomeNexusImpl;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
-//import org.springframework.test.util.ReflectionTestUtils;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
-@ContextConfiguration(classes=MockGenomeNexusImpl.class)
-@RunWith(SpringJUnit4ClassRunner.class)
-public class MutationRecordTest {
+class MutationRecordTest {
```

Comment:
I have questioned the change of the MutationRecord model above. The semantic change so that the class represents an amorphous tab delimited record with no constraints on the fields present (or their data types / allowable values) would mean a drastic change to the unit tests obviously. Maybe the unit tests which were/are present here can point us at some of the constraints which we used to have and which are being dropped. I never did like the "additionalFields" part of our modeling. I thought that was mushy and unstructured. So getting rid of the weirdness of having an "everything else" set of values is a good thing. But I'm personally uncomfortable with the idea of making all the fields become additionalFields and having no core at all.